tap_db2/sync_strategies/common.py

```diff
@@ -232,28 +237,28 @@ def sync_query(
             md_map = metadata.to_map(catalog_entry.metadata)
             stream_metadata = md_map.get((), {})
             replication_method = stream_metadata.get("replication-method")
-
+        
```

Comment:
Looks like there is a bit of whitespace added to a few of these lines

Reply:
This is now resolved